elliott/elliottlib/cli/find_bugs_golang_cli.py

```
@@ -391,6 +393,11 @@ def _get_cves_from_nvr_changelogs(self) -> Set[str]:
 
         items = list(cves_map.items())
         if not all(v == items[0][1] for _, v in items):
+            if self.skip_golang_cve_consistency_check:
+                LOGGER.warning(
+                    f"Skipping CVE consistency check. Inconsistent CVE IDs in changelogs for fixed in NVRs: {cves_map}"
+                )
+                return list(cves_map.values())[0]
```

Ruff (0.16.1)

[warning] Prefer `next(iter(cves_map.values()))` over single element slice

Replace with `next(iter(cves_map.values()))`

(RUF015)